src/ontology/mondo-edit.obo

```diff
@@ -484878,21 +484878,17 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/1607/crawfur
 
 [Term]
 id: MONDO:0022900
-name: athyreotic congenital hypothyroidism
-def: "Congenital hypothyroidism in which fetal deficiency was severe because of complete absence (athyreosis) of the gland. Physical features may include a larger anterior fontanel, persistence of a posterior fontanel, an umbilical hernia, and a large tongue (macroglossia)" [Wikipedia:Congenital_hypothyroidism]
+name: obsolete athyreotic congenital hypothyroidism
+def: "OBSOLETE. Congenital hypothyroidism in which fetal deficiency was severe because of complete absence (athyreosis) of the gland. Physical features may include a larger anterior fontanel, persistence of a posterior fontanel, an umbilical hernia, and a large tongue (macroglossia)" [Wikipedia:Congenital_hypothyroidism]
 comment: Reason of obsoletion: out of scope - MONDO:excludeHistoricalDisease. Term to consider: athyreosis-MONDO:0019855
```

Comment:
the comment can be removed since all the information is in annotations. It is not crucial though.

Reply:
@sabrinatoro I was following optional step 11 in the documentation :) I can remove it though if it unnecessary.

Reply:
haha! Indeed, in our document, we say that a comment is helpful, which is why I said "it is not crucial" :-) 
Thank you for following the documentation though !